tests/test_census_gazetteer.py

```
@@ -0,0 +1,131 @@
+    assert result.country == "US"
+    assert result.admin_levels["state_fips"] == "01"
+    assert result.admin_levels["county_fips"] == "073"
+    assert result.geometry.geom_type == "Polygon"
+
+
+def test_lookup_backend_error_on_500(census_gaz):
+    from siege_utilities.geo.gazetteers.base import GazetteerBackendError
+
+    bad = MagicMock(status_code=500, text="upstream error")
+    with patch.object(census_gaz._session, "get", return_value=bad):
+        with pytest.raises(GazetteerBackendError, match="500"):
```

Ruff (0.15.12)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)